src/passes/MemoryPacking.cpp

```diff
@@ -120,6 +124,28 @@ struct MemoryPacking : public Pass {
     }
     module->memory.segments.swap(packed);
   }
+
+  void optimizeTrappingBulkMemoryOps(Module* module) {
+    struct Trapper : PostWalker<Trapper> {
```

Comment:
it's not urgent but this should be parallelized. please add at least a TODO about that